### ISSUE TITLE
Android crash when using SymbolLayer release build only

### ISSUE THREAD

**Issue by briancalvium** [nitaliano/react-native-mapbox-gl#857]
```
java.lang.NullPointerException · Attempt to invoke virtual method 'android.graphics.Bitmap$Config android.graphics.Bitmap.getConfig()' on a null object reference

NativeMapView.java:822com.mapbox.mapboxsdk.maps.NativeMapView.addImage	
MapboxMap.java:454com.mapbox.mapboxsdk.maps.MapboxMap.addImage	
DownloadMapImageTask.java:61com.mapbox.rctmgl.utils.DownloadMapImageTask.onPostExecute	
DownloadMapImageTask.java:22com.mapbox.rctmgl.utils.DownloadMapImageTask.onPostExecute	
AsyncTask.java:660android.os.AsyncTask.finish
```

This works on iOS (always) and Android dev in the simulator

Here is the setup:
```
const MAP_PIN_IMAGE = require('../assets/images/pin.png');
...

const mapLayerStyles = MapboxGL.StyleSheet.create({
  icons: {
    iconImage: '{icon}',
    iconAllowOverlap: true,
    iconSize: MapboxGL.StyleSheet.source(
      {
        item: 1,
      },
      'icon',
      MapboxGL.InterpolationMode.Categorial
    ),
  },
});

...

const featureCollection = {
      type: 'FeatureCollection',
      features: [
        {
          type: 'Feature',
          id: 'item-feature',
          properties: {icon: 'item'},
          geometry: {type: 'Point', coordinates: [itemCoordinates.longitude, itemCoordinates.latitude]},
        },
      ],
    };

    return (
      <MapboxGL.ShapeSource id={'shape-source'} shape={featureCollection} images={{item: MAP_PIN_IMAGE}}>
        <MapboxGL.SymbolLayer id="item-symbol" style={mapLayerStyles.icons} />
      </MapboxGL.ShapeSource>
    );

```

It seems suspicious that it is trying to download the image? Or is that "downloading" from the RN bundle?

**Comment by nitaliano** [comment]
It's just the name of the class of the AsyncTask pulling in the image, it's given a url to the file so (https://, http://, file://, ...)  it jumps to a background thread to open up a stream and put the image into a bitmap, then back to the main thread where we register that bitmap with GL Native.

I'll run something in release mode today and see what happens. The size of the Bitmap could also be addressed by messing with the Bitmap options on the native side.

@briancalvium also forgot to mention that if you upload your image through Mapbox studio as an SVG it will already exist on the Map and the sizing will be correct when you reference it, just as a workaround for now

**Comment by briancalvium** [comment]
As a workaround, I've put the image in drawable and access it using `{assets: ['pin']}` on Android, but it would be great to be able to just do the same thing on both. Thanks for the help!

**Comment by nitaliano** [comment]
If you put your image in `Images.xcassets` you'll be able to also reference that with assets on iOS. The plus side of this is you can put in images based on pixel ratios on each platform so the icons will look good on every screen

**Comment by phenbach** [comment]
@briancalvium @nitaliano I am experiencing this same issue. Can you please give more details of the work around?

**Comment by nitaliano** [comment]
I'm in the process of fixing this, I have it working on Android now and I'm hoping a PR will drop for it at the end of the week.

@phenbach it's not really a work around it's a preferences that just happens to solve this issue and if you're used to native development you might be more comfortable having your native projects control your assets....so on your `ShapeSource` you can register what images you want to use, so if you store your images in `Images.xcassets` on iOS and `drawables` on Android, we will pull the images out of there. The example application does this, if you have any questions let me know!

**Comment by phenbach** [comment]
@nitaliano Thanks for responding. We worked it out by adding the images to the android assets and referencing the drawables

**Comment by nitaliano** [comment]
PR is out that fixes this issue https://github.com/mapbox/react-native-mapbox-gl/pull/922 sorry for the inconvenience

**Comment by nitaliano** [comment]
Fix has been merged into master will be published in `6.0.3` if you're still having issues with this feel free to reopen
